pulp_python/pytest_plugin.py

```diff
@@ -0,0 +1,194 @@
+import pytest
+import uuid
+import subprocess
+
+from pulpcore.tests.functional.utils import BindingsNamespace
+from pulp_python.tests.functional.constants import (
+    PYTHON_FIXTURE_URL,
+    PYTHON_XS_PROJECT_SPECIFIER,
+    PYTHON_EGG_FILENAME,
+    PYTHON_URL,
+)
+
+
+# Bindings API Fixtures
+
+
+@pytest.fixture(scope="session")
+def python_bindings(_api_client_set, bindings_cfg):
+    """
+    A namespace providing preconfigured pulp_python api clients.
+
+    e.g. `python_bindings.RepositoriesPythonApi.list()`.
+    """
+    from pulpcore.client import pulp_python as python_bindings_module
+
+    python_client = python_bindings_module.ApiClient(bindings_cfg)
```

Comment:
I started to call this "api_client" to allow for easier copy and paste...
(Did i just say that? :hand_over_mouth: )